react-backend/routes/cbauthentication/CBPasswordResetInstigator.js

```diff
@@ -20,30 +20,34 @@ router.post('/', (req, res, next) => {
     if (data.formEmail.length === 0) {
       res.send('noinput');
     } else if (validator.isEmail(data.formEmail)) {
-      getCBAlreadyExists(data.formEmail, (error, result) => {
-        if (error) {
-          console.log('error from getCBAlreadyExists ', error);
-          res.status(500).send({
-            error: 'Cannot access database to check if cbemail exists'
-          });
-        } else {
+      getCBAlreadyExists(data.formEmail)
+        .then((result) => {
           res.send(result.rows[0].exists);
           if (result.rows[0].exists) {
```

Comment:
This and the line above should now be just result. Change the variable name as below.